extensions/ql-vscode/src/databases.ts

```diff
@@ -470,6 +475,25 @@ export class DatabaseItemImpl implements DatabaseItem {
     return uri.scheme === zipArchiveScheme &&
       decodeSourceArchiveUri(uri).sourceArchiveZipPath === this.sourceArchive.fsPath;
   }
+
+  public async isAffectedByTest(testPath: string): Promise<boolean> {
+    const databasePath = this.databaseUri.fsPath;
+    if (!databasePath.endsWith('.testproj')) {
+      return Promise.resolve(false);
```

Comment:
javascript will hoist the result into a promise, so there's no need to explicitly call resolve here.

```suggestion
      return false;
```

Same as below.